standalone-mode/docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+version: "3.5"
```

Comment:
I think the folder name `standalone-mode` which contains this sample a bit too abstract. What do you think about renaming it to `cluster-standalone-mode` or something of that extent so that we know which product it concers?

Reply:
I agree. Thanks. Fixed in 5f5bb05ce974d8bb0a1aad055a6e100d27343dc6. Please take a look again when you have time!